internals/secrethub/tpl/internal/token/token.go

```diff
@@ -0,0 +1,42 @@
+package token
+
+// Tokens
+var (
+	Dollar    = '$'
+	LBracket  = '{'
+	RBracket  = '}'
+	Backslash = '\\'
+
+	tokens = map[rune]struct{}{
+		Dollar:    {},
+		LBracket:  {},
+		RBracket:  {},
+		Backslash: {},
+	}
+)
+
+// IsDollar returns whether the given rune is the dollar token.
+func IsDollar(ch rune) bool {
```

Comment:
We discussed that these functions in itself don't really add any value because ` == ` is just as simple.